src/main/java/org/mastodon/mamut/feature/spot/SpotBranchSpotIDFeature.java

```diff
@@ -0,0 +1,86 @@
+package org.mastodon.mamut.feature.spot;
+
+import org.mastodon.feature.Dimension;
+import org.mastodon.feature.Feature;
+import org.mastodon.feature.FeatureProjection;
+import org.mastodon.feature.FeatureProjectionKey;
+import org.mastodon.feature.FeatureProjectionSpec;
+import org.mastodon.feature.FeatureProjections;
+import org.mastodon.feature.FeatureSpec;
+import org.mastodon.feature.IntFeatureProjection;
+import org.mastodon.feature.Multiplicity;
+import org.mastodon.mamut.model.Spot;
+import org.mastodon.properties.IntPropertyMap;
+import org.scijava.plugin.Plugin;
+
+import java.util.Collections;
+import java.util.Set;
+
+import static org.mastodon.feature.FeatureProjectionKey.key;
+
+public class SpotBranchSpotIDFeature implements Feature< Spot >
```

Comment:
Maybe rename to `SpotBranchIDFeature`? I would find that name a little less confusing.